.github/workflows/deploy-infrastructure.yaml

```diff
@@ -21,6 +21,11 @@ on:
           required: false
           default: ""
           type: string
+      multi_domain:
+        description: "Domain to assume github runner role "
```

Comment:
Maybe a better description - eg If true uses a multi domain github runner ; if false uses github runner for triggering repo ... or similar

Reply:
Description updated